public/locales/en-US/translations.json

```diff
@@ -160,6 +166,7 @@
   "transaction_EnableAmendment": "Enable Amendment",
   "transaction_SetFee": "Set Fee",
   "transaction_UNLModify": "UNL Modify",
+  "pseudo_transactions":"Pseudo Transactions",
```

Comment:
nit: shouldn't this be with the categories above?

Reply:
fixed